examples/births-deaths/population_manager.rs

```diff
@@ -0,0 +1,353 @@
+use crate::parameters_loader::Parameters;
+use ixa::context::Context;
+use ixa::global_properties::ContextGlobalPropertiesExt;
+use ixa::people::{ContextPeopleExt, PersonId, PersonProperty};
+use ixa::random::define_rng;
+use ixa::random::ContextRandomExt;
+use ixa::{define_global_property, define_person_property, define_person_property_with_default};
+use rand_distr::Uniform;
+use serde::Deserialize;
+use std::collections::HashMap;
+
+define_rng!(PeopleRng);
+
+static MAX_AGE: f64 = 100.0;
+use rand_distr::Exp;
+use serde::Serialize;
+use std::fmt;
+
+#[derive(Debug, Hash, Eq, PartialEq, Clone, Copy, Serialize, Deserialize)]
+pub enum InfectionStatus {
+    S,
+    I,
+    R,
+}
+
+#[derive(Deserialize, Serialize, Copy, Clone, PartialEq, Eq, Debug, Hash)]
+pub enum AgeGroupRisk {
+    NewBorn,
+    General,
+    OldAdult,
+}
+
+impl fmt::Display for AgeGroupRisk {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{self:?}")
+    }
+}
+
+define_global_property!(Foi, HashMap<AgeGroupRisk, f64>);
+define_person_property_with_default!(InfectionStatusType, InfectionStatus, InfectionStatus::S);
+
+define_person_property!(Birth, f64);
+define_person_property!(Alive, bool);
+
+fn schedule_birth(context: &mut Context) {
+    let parameters = context.get_global_property_value(Parameters).clone();
+    let _person = context.create_new_person(context.get_current_time());
+
+    let next_birth_event = context.get_current_time()
+        + context.sample_distr(PeopleRng, Exp::new(parameters.birth_rate).unwrap());
+    context.add_plan(next_birth_event, move |context| {
+        schedule_birth(context);
+    });
+}
+
+fn schedule_death(context: &mut Context) {
+    let parameters = context.get_global_property_value(Parameters).clone();
+
+    if let Some(person) = context.sample_person_by_property(Alive, true) {
+        context.attempt_death(person);
+
+        let next_death_event = context.get_current_time()
+            + context.sample_distr(PeopleRng, Exp::new(parameters.death_rate).unwrap());
+
+        context.add_plan(next_death_event, move |context| {
+            schedule_death(context);
+        });
+    }
+    // Cancel all plans
```

Comment:
Did you mean to leave this as TODO?

Reply:
Not really. I think I realized later that I don't actually know who should take care of canceling all plans. I ended up implementing it in `infection_manager` and just registering for death events. I think that approach isn't too bad, so I just forgot to remove from population manager.

Reply:
Ok thanks. Yeah, just remove the comment then